drivers/snmp-ups.c

```diff
@@ -2294,9 +2294,6 @@ static int base_snmp_template_index(const snmp_info_t *su_info_p)
 	char test_OID[SU_INFOSIZE];
 	snmp_info_flags_t template_type = get_template_type(su_info_p->info_type);
 
-	if (!su_info_p->OID)
```

Comment:
Can't this cause a null pointer deref just below?

Reply:
Adjusted just to be sure. I thought the block was moved up by me recently to avoid the null in the first place, but seems it was a similar piece of code in another driver.